website/docs/index.html.markdown

```diff
@@ -16,6 +16,9 @@ Pricing and signup information can be found at https://www.fastly.com/signup
 
 Use the navigation to the left to read about the available resources.
 
+The Fastly provider prior to version xxx requires using [--parallelism=1]
+(/docs/commands/apply.html#parallelism-n) for `apply` operations.
```

Comment:
I have looked at all the other commits for the upcoming release and I think we should go with `v0.13.0`.